### PR TITLE
Add last two config variables, JSON config for questions

### LINKED ISSUE #874: Better community-facing documentation
As a {Product Owner}, I want the community team to write better docs for managing a survey + index, because the current ones are not great.

The technical documentation has been updated, but we need better, consolidated community-facing documentation.

Directly related to #808 and #813

### LINKED ISSUE #876: Review and document Question config objects
Related to #842.

Review the current QuestionSet and Question json config objects as used in the spreadsheet CMS, to ensure internal consistency and add uptodate json schemas. Document these structures so site users can understand and customise their own Questions and QuestionSets.

This is partly documented in the [MigrationNotes.md](https://github.com/okfn/opendatasurvey/blob/25b99e3d72c1d8b527c5667920620590729e35d8/MigrationNotes.md) document.

### PR DESCRIPTION
**Pull request by tlacoyodefrijol**
Closes #874. Closes #876.

### CODE REVIEW

File: docs/site-admins/index.md [hunk 1]
```diff
@@ -326,6 +340,233 @@ absent, the `discussionForum` global settings will be used instead.
 
 A URL to a forum where users can discuss pending submissions. This URL will be linked from the bottom of the pending submission page. If the URL links to the discuss.okfn.org Discourse instance (in the format `http://discuss.okfn.org/c/<topic>/<subtopic>`), the link will be formatted to automatically create a new topic, pre-populated with a title, category, and body text linking back to the submission.
 
+ ## Questions and Question Sets
+
+Questions are loaded to the Survey through the `Questions` and `QuestionSet`sheets in the CMS. If you make changes to the `QuestionSet`values, make sure they validate against the JSON schema before reloading your survey.
```
Comment (brew) [inline docs/site-admins/index.md]: Spaces needed after ```"`QuestionSet`sheets"``` and ```"`QuestionSet`values"```.

File: docs/site-admins/index.md [hunk 2]
```diff
@@ -326,6 +340,233 @@ absent, the `discussionForum` global settings will be used instead.
 
 A URL to a forum where users can discuss pending submissions. This URL will be linked from the bottom of the pending submission page. If the URL links to the discuss.okfn.org Discourse instance (in the format `http://discuss.okfn.org/c/<topic>/<subtopic>`), the link will be formatted to automatically create a new topic, pre-populated with a title, category, and body text linking back to the submission.
 
+ ## Questions and Question Sets
+
+Questions are loaded to the Survey through the `Questions` and `QuestionSet`sheets in the CMS. If you make changes to the `QuestionSet`values, make sure they validate against the JSON schema before reloading your survey. 
+
+In the old Census, Questions were associated with Datasets at the site level, i.e. one set of Questions was used for all Datasets. This is no longer a restriction. Questions are grouped in QuestionSets, and each site can configure one QuestionSet for use by all of the site's Datasets, or a QuestionSet can be defined for each Dataset individually.
```
Comment (brew) [inline docs/site-admins/index.md]: This is technically true, but we don't currently support it well on the front-end. I'd remove this.

File: docs/site-admins/index.md [hunk 3]
```diff
@@ -326,6 +340,233 @@ absent, the `discussionForum` global settings will be used instead.
...
+
+In the CMS, the Datasets sheet has a new `QuestionSetURL` column. The value is the url pointing to the `QuestionSet` config sheet. A site-wide Question Set url can be added to the site config page under the key `question_set_url`. A further system-wide fallback Question Set url can be set as an env var, `FALLBACK_QUESTIONSET` (this is provided for backward-compatibility to smooth migration to the new system).
```
Comment (brew) [inline docs/site-admins/index.md]: Again, I'd remove the first sentence which refers to a Dataset-specific QuestionSetURL, and leave the other bit about the site-wide url (which we do support).

File: docs/site-admins/index.md (hunk 2 above)
Comment (brew) [inline docs/site-admins/index.md]: Specifically:
> In the old Census, Questions were associated with Datasets at the site level, i.e. one set of Questions was used for all Datasets. This is no longer a restriction. Questions are grouped in QuestionSets, and each site can configure one QuestionSet for use by all of the site's Datasets, or a QuestionSet can be defined for each Dataset individually.

File: docs/site-admins/index.md (hunk 3 above)
Comment (brew) [inline docs/site-admins/index.md]: Specifically:

> In the CMS, the Datasets sheet has a new `QuestionSetURL` column. The value is the url pointing to the `QuestionSet` config sheet.

**Comment by morchickit** [comment]
@tlacoyodefrijol any updates on this?

**Pull request merged**